RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -2,33 +2,41 @@
 @using System.Globalization
 @inject IBillRepository billRepository
 
-@section Styles {
-<link rel="stylesheet" href="/css/BillDetails.css" />
-}
 @{
     var billId = Guid.Parse(RouteData.Values["guid"].ToString());
     if (billRepository.GetById(billId) is null)
     {
-        <div id="errorText" style="color:red"> Bill was not found!</div>
+        <div class="alert alert-danger text-center">
+            Bill was not found!
+        </div>
     }
     else
     {
+        <h1 class="text-center">Bill Details</h1>
         var item = billRepository.GetById(billId);
-        <div class="bill-summary">
-            <div t-id="description">@item.Description</div>
-            <div t-id="personWhoPaid">Paid by: @(item.EntryLines[0].Person.FirstName)
-                @(item.EntryLines[0].Person.LastName) </div>
-            <div t-id="date">@item.Date.ToString("D")</div>
-            <div t-id="amount">Total amount: @item.EntryLines[0].Amount€</div>
+        <div class="p-3 border rounded mx-auto" style ="background-color: #f7f7f7">
+            <dl class="row">
+                <dt class="col-12 col-sm-6">Description:</dt>
+                <dd t-id="description" class="col-12 col-sm-6 text-sm-end">@item.Description</dd>
+
+                <dt class="col-12 col-sm-6">Paid by:</dt>
+                <dd t-id="personWhoPaid" class="col-12 col-sm-6 text-sm-end">@item.EntryLines[0].Person.FirstName @item.EntryLines[0].Person.LastName</dd>
+
+                <dt class="col-12 col-sm-6">Date:</dt>
+                <dd t-id="date" class="col-12 col-sm-6 text-sm-end">@item.Date.ToString("D")</dd>
+
+                <dt class="col-12 col-sm-6">Total amount:</dt>
+                <dd t-id="amount" class="col-12 col-sm-6 text-sm-end">@item.EntryLines[0].Amount€</dd>
+            </dl>
         </div>
-        <div class="bill">
+
+        <div class="p-3 border rounded mx-auto bg-white">
             @for (int i = 1; i < item.EntryLines.Length; i++)
             {
-                <div t-id="entryLine">
-                    <div t-id="person">@(item.EntryLines[i].Person.FirstName)
-                        @(item.EntryLines[i].Person.LastName): </div>
-                    <div t-id="amountPerPerson">@(-item.EntryLines[i].Amount)€ </div>
-                </div>
+                <dl t-id="entryLine" class="row">
+                    <dt class="col-12 col-sm-6" t-id="person">@item.EntryLines[i].Person.FirstName @item.EntryLines[i].Person.LastName:</dt>
+                    <dd t-id="amountPerPerson" class="col-12 col-sm-6 text-sm-end">@(-item.EntryLines[i].Amount)€</dd>
```

Comment:
is this printing a literal minus sign regardless of the amount value being positive or negative? 

If my assumption is correct:
this isn't releated to the current task so should be separate ticket but I believe this is a bug in some scenarios.

Reply:
I made this adjustment, since the entries for the people whose meals were paid for show negative amounts, and we only want to display the amounts without any signs. There is no literal minus sign printed, but the value is inverted, turning negative numbers into positive ones.

Reply:
Ah, I see. would you rather use https://learn.microsoft.com/en-us/dotnet/api/system.math.abs instead to document you want to show the absolute value instead of a signed value? Of course, this hides positive payments. 

If the Math.Abs version is not correct, I would recommend uing `*-1` (multiply by minus one) instead of prefixing with minus because it's more obvious when reading the code that you are inverting the sign.